packages/trezor-user-env-link/src/websocket-client.ts

```diff
@@ -241,7 +262,40 @@ export class TrezorUserEnvLink extends EventEmitter {
 
         this.removeAllListeners();
     }
+
+    waitForTrezorUserEnv() {
+        return new Promise<void>(async (resolve, reject) => {
+            // unfortunately, it can take incredibly long for trezor-user-env to start, we should
+            // do something about it
+            const limit = 300;
+            let error = '';
+
+            // eslint-disable-next-line no-console
+            console.log('waiting for trezor-user-env');
+
+            for (let i = 0; i < limit; i++) {
+                if (i === limit - 1) {
+                    // eslint-disable-next-line no-console
+                    console.log(`cant connect to trezor-user-env: ${error}\n`);
+                }
+                await delay(1000);
+
+                try {
+                    const res = await fetch('http://localhost:9002');
+                    if (res.status === 200) {
+                        return resolve();
+                    }
+                } catch (err) {
+                    error = err.message;
```

Comment:
This function is logging `waiting for trezor-user-env`  when it starts waiting but there is no log when it successfully connect to trezor-user-env. 
I think having extra log when the connection is successful could be useful to know when running test if waiting for trezor-user-env for the tests.

Reply:
[4a0f826](https://github.com/trezor/trezor-suite/pull/6436/commits/4a0f826d64b38c9690bd2a811020682507006955)